sonar-db/src/main/java/org/sonar/db/version/DatabaseVersion.java

```diff
@@ -37,7 +37,7 @@
    * versions must be previously upgraded to LTS version.
    * Note that the value can't be less than current LTS version.
    */
-  public static final int MIN_UPGRADE_VERSION = 600;
+  public static final int MIN_UPGRADE_VERSION = 1152;
```

Comment:
Installations upgrading from 1150 without having 1152 sounds a very weird corner-case. Probably the upgrade to 6.x would fail. But still, keeping 1152 is ok.